src/js/components/ServiceForm.js

```diff
@@ -23,7 +23,7 @@ const DUPLICABLE_FIELDS_TO_WATCH = {
   },
   ports: {
     fields: ['discovery', 'protocol'],
-    blurOnly: ['name', 'lbPort'],
+    blurOnly: ['name'],
```

Comment:
Could you add a short explanation as to why we don't need `lbPort` in this array? To be honest I don't really know what it's doing anyway...